docs/source/howto/installation.rst

```diff
@@ -547,20 +547,30 @@ See the :doc:`../reference/_changelog` for a list of breaking changes.
 
 .. _how-to:installation:backup:
 
-Backing up your installation
+Backing up your data
 ============================
 
-A full backup of an AiiDA instance and AiiDA managed data requires a backup of:
+General information
+-----------------
 
-* the AiiDA configuration folder, which is named ``.aiida``.
-  The location of the folder is shown in the output of ``verdi status``.
-  This folder contains, among other things, the ``config.json`` configuration file and log files.
+The most convenient way to back up the data of a single AiiDA profile is to use
 
-* the data stored for each profile.
-  Where the data is stored, depends on the storage backend used by each profile.
+.. code:: bash
 
-The panels below provide instructions for storage backends provided by ``aiida-core``.
-To determine what storage backend a profile uses, call ``verdi profile show``.
+    $ verdi --profile <profile_name> storage backup /path/to/destination
+
+This command is safe to use when AiiDA is running, it automatically manages a subfolder structure of previous backups, and new backups are done in an efficient way (using ``rsync`` hard-link functionality to the previous backup).
```

Comment:
I guess this is true for the implementation of `core.psql_dos`, but I don't think we can state this in general. We should change this to say something like:
```
Whether it is safe to run the backup operation while AiiDA is running, depends on the specific storage backend used by the profile. Please check with the instructions of the storage backend. To err on the safe side, only perform a backup when the profile is not in use.
```

Reply:
I can add this caveat for now (and remove the same "safe" claim from the CLI docs), but I think we probably would want to make this command as safe to run as possible (even during e.g. a high-throughput run) also for the other storage backends.

Reply:
> but I think we probably would want to make this command as safe to run as possible (even during e.g. a high-throughput run) also for the other storage backends.

Sure, we can do this for the storages that `aiida-core` ships, but mind that `verdi storage backup` can be used with _any_ storage, also those provided by plugins, and we cannot enforce that safety there.